cloud/endagaweb/templates/dashboard/network_detail/edit.html

```diff
@@ -41,7 +41,7 @@
 
 <div class = 'row'>
   {% include "dashboard/network_detail/nav.html" with active_tab='edit'%}
-
+  {% if 'endagaweb.edit_network' in user_permission %}
```

Comment:
During creation of a any user (like: cloud admin) who can have multiple networks we are currently giving same permissions for all networks assigned to it.
Other users (like: loader, Business Analyst etc) are not having multiple n/w access.
 
eg: In current implementation if a cloud admin assigned to network_1 and network_2 it will have same permissions for both.